apps/namadillo/src/atoms/integrations/functions.ts

```diff
@@ -170,6 +170,15 @@ const tryDenomToIbcAsset = async (
 
   const { path, baseDenom } = denomTrace;
 
+  const assetOnRegistry = tryDenomToRegistryAsset(
```

Comment:
Maybe you already accounted for this, but there are cases where the same asset will have different configurations on different chains, so doing this allows more assets to be decoded but at the risk of showing the wrong information for an asset (which would mainly matter if the exponents are different, since we could be showing the wrong amount).